### PR TITLE
Use "projectId" instead of "project_id" + 404 status update + remove status capitalization

### PR DESCRIPTION
**Pull request by fjaguero**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -65,21 +65,19 @@ You can handle your available classifiers in your [Console](https://console.lang
...
- "project_id": "Your project id"
+ "projectId": "Your project id"
 }
 ```
 
 Parameter | Required | Default | Type | Description
 --------- | -------  | ------- | ---- | -----------
 text      | true     | -       | string | The text to be analyzed.
-project_id | true | -       | string | The project to be used to analyze the text.
+projectId | true | -       | string | The project to be used to analyze the text.
```
Comment (albertorestifo) [inline source/index.html.md]: Can we align this table? That way I'll be easier to read.

**Review by albertorestifo — CHANGES_REQUESTED** [review]

File: source/index.html.md (hunk 1 above)
Comment (fjaguero) [reply source/index.html.md]: Fixed

**Review by albertorestifo — APPROVED** [review]
Perfect, thanks! 👍

**Pull request merged**